### ISSUE TITLE
It seems I experience calls to get executed in line and not threaded like any webserver would

### ISSUE THREAD

**Issue by Nil0** [bakercp/ofxJSONRPC#13]
I have a website calling an openFrameworks_v0.9.8 app via ofxjsonrpc. Everything worked fine (not that many calls) until I had to implement som feedback communication from a udp based unit. In this case I test 5 concurrent calls via ajax that all takes one second and I notice that they get executed in line. Additionally all other automated calls I got at the same time can never come in between those 5 calls only before or delayed until after they are done.

**Comment by bakercp** [comment]
Yes, this is a currently limitation of the implementation since calls hook back into the main event loop.
